[16.042709351] 2019-12-29 12:14:30,258 - INFO - launcher: Starting sanity checks
[16.042709351] 2019-12-29 12:14:30,258 - INFO - launcher: Checking for Python 3.5+
[16.544342041] 2019-12-29 12:14:30,259 - INFO - launcher: Checking console encoding
[16.544342041] 2019-12-29 12:14:30,259 - INFO - launcher: Setting console encoding to UTF-8
[16.544342041] 2019-12-29 12:14:30,259 - INFO - launcher: Ensuring we're in the right environment
[17.045021057] 2019-12-29 12:14:30,259 - INFO - launcher: Adding local bins/ folder to path
[282.251119614] 2019-12-29 12:14:30,524 - INFO - launcher: Required checks passed.
[282.752275467] 2019-12-29 12:14:30,525 - INFO - launcher: Optional checks passed.
[282.752275467] 2019-12-29 12:14:30,525 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[0.6066143512725830] 2019-12-29 12:14:30,849 - WARNING - musicbot.config | In config.py::MainThread(6432), line 121 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[0.6066143512725830] 2019-12-29 12:14:30,849 - INFO - musicbot.config | In config.py::MainThread(6432), line 132 in run_checks: Using i18n: config/i18n/en.json
[0.6081180572509766] 2019-12-29 12:14:30,850 - DEBUG - musicbot.bot | In bot.py::MainThread(6432), line 225 in _setup_logging: Set logging level to INFO
[0.6091208457946777] 2019-12-29 12:14:30,851 - DEBUG - musicbot.json | In json.py::MainThread(6432), line 8 in __init__: Init JSON obj with config/i18n/en.json
[0.6201498508453369] 2019-12-29 12:14:30,862 - INFO - musicbot.bot | In bot.py::MainThread(6432), line 90 in __init__: Starting MusicBot release-260819-4-gf7d0d74
[0.6206512451171875] 2019-12-29 12:14:30,863 - INFO - musicbot.bot | In bot.py::MainThread(6432), line 96 in __init__: Loaded autoplaylist with 2531 entries
[3.8748085498809814] 2019-12-29 12:14:34,117 - DEBUG - musicbot.bot | In bot.py::MainThread(6432), line 951 in on_ready: Connection established, ready to go.
[3.8748085498809814] 2019-12-29 12:14:34,117 - DEBUG - musicbot.bot | In bot.py::MainThread(6432), line 342 in _cache_app_info: Caching app info
[4.1450281143188477] 2019-12-29 12:14:34,387 - DEBUG - musicbot.bot | In bot.py::MainThread(6432), line 773 in _scheck_ensure_env: Ensuring data folders exist
[4.1455292701721191] 2019-12-29 12:14:34,388 - DEBUG - musicbot.bot | In bot.py::MainThread(6432), line 789 in _scheck_server_permissions: Checking server permissions
[4.1455292701721191] 2019-12-29 12:14:34,388 - DEBUG - musicbot.bot | In bot.py::MainThread(6432), line 793 in _scheck_autoplaylist: Auditing autoplaylist
[4.1460309028625488] 2019-12-29 12:14:34,388 - DEBUG - musicbot.bot | In bot.py::MainThread(6432), line 797 in _scheck_configs: Validating config
[4.1460309028625488] 2019-12-29 12:14:34,388 - DEBUG - musicbot.config | In config.py::MainThread(6432), line 236 in async_validate: Validating options...
[4.1460309028625488] 2019-12-29 12:14:34,388 - DEBUG - musicbot.config | In config.py::MainThread(6432), line 250 in async_validate: Acquired owner id via API
[4.1460309028625488] 2019-12-29 12:14:34,388 - DEBUG - musicbot.bot | In bot.py::MainThread(6432), line 800 in _scheck_configs: Validating permissions config
[4.1460309028625488] 2019-12-29 12:14:34,388 - DEBUG - musicbot.permissions | In permissions.py::MainThread(6432), line 94 in async_validate: Validating permissions...
[4.1460309028625488] 2019-12-29 12:14:34,388 - DEBUG - musicbot.permissions | In permissions.py::MainThread(6432), line 98 in async_validate: Fixing automatic owner group
[4.1460309028625488] 2019-12-29 12:14:34,388 - INFO - musicbot.bot | In bot.py::MainThread(6432), line 968 in on_ready: Connected: 656383005740367873/Miku#9970
[4.1465320587158203] 2019-12-29 12:14:34,389 - INFO - musicbot.bot | In bot.py::MainThread(6432), line 976 in on_ready: Owner:     562915923460358164/AI_Spipe#9191

[4.1470332145690918] 2019-12-29 12:14:34,389 - INFO - musicbot.bot | In bot.py::MainThread(6432), line 979 in on_ready: Guild List:
[4.1470332145690918] 2019-12-29 12:14:34,389 - INFO - musicbot.bot | In bot.py::MainThread(6432), line 983 in on_ready:  - ¦-¦ `s Server
[4.1475346088409424] 2019-12-29 12:14:34,390 - INFO - musicbot.bot | In bot.py::MainThread(6432), line 1039 in on_ready: Not bound to any text channels
[4.1480360031127930] 2019-12-29 12:14:34,390 - INFO - musicbot.bot | In bot.py::MainThread(6432), line 1065 in on_ready: Not autojoining any voice channels
